source-code/ide-extension/package.json

```diff
@@ -19,7 +19,11 @@
 		"Linters",
 		"Visualization"
 	],
-	"main": "./dist/main.cjs",
+	"exports": {
+		".": "./dist/ide/main.cjs",
+		"./plugin": "./dist/plugin/index.js"
+	},
```

Comment:
no exports are needed and might confuse vscode (we are not using npm for plugins and vscode only uses main)